create_android_makefiles

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Run this script ONLY inside an Android build system
+# and after you ran lunch command!
+
+if [ -z "$ANDROID_BUILD_TOP" ]; then
+	echo "Run lunch before running this script!"
+	exit 1
+fi
+
+if [ -z "$1" ]; then
+	ARCH="arm"
+else
+	ARCH="$1"
+fi
+
+if [ $ARCH = "x86" ]; then
+	TARGET_ARCH="ia32"
+else
+	TARGET_ARCH="$ARCH"
+fi
+
+./configure \
+	--without-snapshot \
+	--openssl-no-asm \
+	--dest-cpu=$TARGET_ARCH \
+	--dest-os=android
+
+export GYP_GENERATORS="android"
+export GYP_GENERATOR_FLAGS="limit_to_target_all=true"
+export GYP_DEFINES="target_arch=$TARGET_ARCH v8_target_arch=$TARGET_ARCH android_target_arch=$ARCH host_os=linux OS=android"
```

Comment:
Style issue: can you break up this line so it's <= 80 columns?  Also, tabs => 2 spaces.

Can you move this file to e.g. the tools/ directory?  I'd like to avoid cluttering the top-level directory with another file.